### ISSUE TITLE
502 Proxy Error - DNS lookup failure for: php

### ISSUE THREAD

**Issue by tracyfloyd** [cytopia/devilbox#99]
After updating to latest devilbox I am getting a 502 Proxy Error for all my .loc domains (and for http://localhost) that use php. I setup a .loc with just an index.html file in it and it works fine.
#
**Browser page reads:**
```
The proxy server received an invalid response from an upstream server.
The proxy server could not handle the request GET /index.php.

Reason: DNS lookup failure for: php
```

**System info:**
```
OS: Mac OSX Sierra (10.12.5)
docker version: 17.06.0-ce, build 02c1d87
docker-compose version: 1.14.0, build c7bdf9e
Using: apache-2.4,  mysql-5.7,  php-fpm-7.0
```

**log/apache-2.4/devilbox-error.log:**
```
[Fri Jul 21 13:52:11.121779 2017] [proxy:error] [pid 92] [client 172.16.238.1:58212] AH00898: DNS lookup failure for: php returned by /index.php
```

**Comment by tracyfloyd** [comment]
Got this working again by changing `NEW_UID` and `NEW_GID` to `1000` in the .env file (instead of my local user's uid and gid as directed in the .env file's comments)

Same as https://github.com/cytopia/devilbox/issues/96

**Comment by cytopia** [comment]
Let's continue the discussion in #96 as it is the same

**Comment by cytopia** [comment]
First fix for PHP 7.0 is on its way:

https://github.com/cytopia/docker-php-fpm-7.0/pull/22
